chroma-manager/tests/feature/cli/features/steps/standard_cli.py

```diff
@@ -3,6 +3,7 @@
 # Copyright (c) 2012 Whamcloud, Inc.  All rights reserved.
 # ========================================================
 
+from __future__ import print_function
```

Comment:
in order to print the debug necessary to diagnose and fix an issue related to the changes in this patch in the behave tests, this had to be added (behave wasn't working with the original print function when the AssertionError was raised). I will update the header